### PR TITLE
Add album art preview to home page

### PR DESCRIPTION
**Pull request by RahulTirumalareddy**

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -14,16 +14,21 @@ <h2 class="post-list-heading">{{ page.list_title | default: "Posts" }}</h2>
...
-        <span class="post-meta">{{ post.date | date: date_format }}</span>
-        <h3>
-          <a class="post-link" href="{{ post.url | relative_url }}">
-            {{ post.title | escape }}
-          </a>
-        </h3>
-        {%- if site.show_excerpts -%}
-          {{ post.content | markdownify | strip_html | truncatewords: 50 }}
-        {%- endif -%}
+        <div class ="entry">
```
Comment (mingyli) [inline _layouts/home.html]: nit: space

File: _layouts/home.html [hunk 2]
```diff
@@ -14,16 +14,21 @@ <h2 class="post-list-heading">{{ page.list_title | default: "Posts" }}</h2>
...
+          <img class="album_art" src={{post.album_art}} />
```
Comment (mingyli) [inline _layouts/home.html:18]: nit: space

File: _posts/2017-10-08-lil-yachty.md [hunk 3]
```diff
@@ -75,7 +76,7 @@ also includes the dollar cost of his brush with the law, which although is
 insignificant compared to prison time, is not nothing.
 
 Throughout *Minnesota* Lil Yachty appeals directly to his listeners.
-In fact, he reprimands “you”, asking 
+In fact, he reprimands “you”, asking
```
Comment (mingyli) [inline _posts/2017-10-08-lil-yachty.md]: hm does your editor automatically remove trailing whitespace?

**Comment by mluogh** [comment]
Sire, I think it would be of further interest if it were such that we locally hosted the images instead of depending on other servers which may go down in the future.

**Review by mingyli — DISMISSED** [review]
Great changes. Couple concerns:

1. loading the images from URL takes a while because they are very large. Maybe we could add smaller thumbnails to our `assets/` directory
2. the page doesn't look that great on mobile anymore. Can we figure out a way to not display images on mobile?
image

File: _posts/2017-10-08-lil-yachty.md (hunk 3 above)
Comment (RahulTirumalareddy) [reply _posts/2017-10-08-lil-yachty.md]: oh yea, looks like Atom was doing that; I added the space back in

**Review by mluogh — APPROVED** [review]
Great work. All of Ming's concerns have been addressed.

**Pull request merged**